files/en-us/web/api/element/requestfullscreen/index.html

```diff
@@ -146,7 +156,24 @@ <h2 id="Examples">Examples</h2>
 <p>You can <a href="https://fullscreen-requestfullscreen-demo.glitch.me/">see this example
     in action</a> or <a
     href="https://glitch.com/edit/#!/fullscreen-requestfullscreen-demo">view or remix the
-    code</a> on <a href="http://glitch.com/">Glitch</a>.</p>
+    code</a> on <a href="https://glitch.com/">Glitch</a>.</p>
+
+<h3>Using navigationUI</h3>
+
+<p>In this example, the entire document is placed into full-screen mode by calling
+  {{DOMxRef("Element.requestFullscreen", "requestFullscreen()")}} on the document's
+  {{DOMxRef("Document.documentElement")}}, which is the document's root
+  {{HTMLElement("html")}} element.</p>
+
+<pre class="brush: js">let elem = document.documentElement;
+
+elem.requestFullscreen({ navigationUI: "show" }).then({}).catch(err =&gt; {
```

Comment:
```suggestion
elem.requestFullscreen({ navigationUI: "show" }).then(() => {}).catch(err =&gt; {
```

From testing `Promise.resolve().then({})` it seems like passing in the object literal `{}` as the "then handler" does work (does nothing) but it doesn't seems like a good idea, and occurs only on this page and in https://developer.mozilla.org/en-US/docs/Web/API/Element/onfullscreenchange#example.